smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/integration/HttpProtocolGeneratorUtils.java

```diff
@@ -248,25 +248,14 @@ static void generateMetadataDeserializer(GenerationContext context, SymbolRefere
     }
 
     /**
-     * Writes a response body stream collector. This function converts the low-level response body stream to
+     * Imports a response body stream collector. This function converts the low-level response body stream to
      * Uint8Array binary data.
      *
      * @param context The generation context.
      */
     static void generateCollectBody(GenerationContext context) {
```

Comment:
Do we need this whole method if it just imports something? Or maybe change the name?

Reply:
I rolled it into the generateCollectBodyString method, which is called in all places that generateCollectBody was called.